packages/theme/src/cli/utilities/theme-uploader.ts

```diff
@@ -18,9 +18,9 @@ type ChecksumWithSize = Checksum & {size: number}
 type FileBatch = ChecksumWithSize[]
 
 // Limits for Bulk Requests
-export const MAX_BATCH_FILE_COUNT = 50
-// 10MB
-export const MAX_BATCH_BYTESIZE = 1024 * 1024 * 10
+export const MAX_BATCH_FILE_COUNT = 20
+// 1MB
+export const MAX_BATCH_BYTESIZE = 1024 * 1024
```

Comment:
Should we add a comment here specifying where is the limit coming from? Or perhaps linking this very PR so the context is not lost?